### DISCUSSION TITLE
Interactive Simulations rather then Videos?

### DISCUSSION THREAD

**Discussion by scherrey** [ManimCommunity/manim#3770] [Suggestions and Proposals]
Just discovered this tool and it look great. However, it seems from what I've read so far that the only output from it is fixed videos. What I'm really interested in is making interactive animations where a viewer can alter/create values in real-time and just experiment with a simulation rather than playing a video. Is that something Manim is capable of or plans on doing eventually? If not is there a good tool for doing this that one might recommend? Thanks!

**Comment by uwezi** [comment]
to some extend this can be done using the opengl renderer, but on the other hand, this is not the intended functionality of Manim and there are plenty of other tools out there which offer interactivity: desmos, geogebra, and of course interactive Jupyter notebooks.

See here for some interactivity
https://youtu.be/KeXBLPC1tns?si=QJ0RTMiWgyvqMTwM
